doc/tutorials/index.md

```diff
@@ -0,0 +1,73 @@
+# Tutorials
+
+The [getting started guides](../getting_started/index.md) aims to get you set up with Panel and introduces you to some of the core concepts. The tutorials on the other hand will take you from being a beginner and guide you through your learning.
+
+---
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`plug;2.5em;sd-mr-1` Param
+:link: param
+:link-type: doc
+
+Panel and other projects in the HoloViz ecosystem all build on Param. In this section you will learn the basic concepts behind Param that you need to know to become an effective user of Panel.
+:::
+
+:::{grid-item-card} {octicon}`container;2.5em;sd-mr-1` Components
+:link: components
+:link-type: doc
+
+Panel is a library that provides a lot of object types and while building an app, even a simple one, you will create and interact with many of them. In this section you will get a high-level overview of the different component types Panel offers and how to use them.
+:::
+
+:::{grid-item-card} {octicon}`pulse;2.5em;sd-mr-1` Interactivity
+:link: interactivity
+:link-type: doc
+
+In this section you learn how to leverage Parameters and dependencies on parameters to add interactivity. In particular we will focus on implementing interactivity through reactivity, rather than the more imperative style of programming you might be used to from other UI frameworks.
+:::
+
+:::{grid-item-card} {octicon}`browser;2.5em;sd-mr-1` Effective Development
+:link: development
+:link-type: doc
+
+In this section we will introduce you to the most important concepts you need to know to become an effective Panel developer in notebooks and your favorite editor.
+:::
+
+:::{grid-item-card} {octicon}`rows;2.5em;sd-mr-1` Layouts
+:link: layouts
+:link-type: doc
+
+In this section we will discover how layouts and sizing works in Panel, taking you through the difference between inherent sizing, fixed sizing and responsive sizing and then cover responsive layouts.
+:::
+
+:::{grid-item-card} {octicon}`code;2.5em;sd-mr-1` Structuring Applications
+:link: structure
+:link-type: doc
+
+In this section we will take you through the process of structuring a more complex application, discussing different approaches for managing reactivity and how to create a composable architecture for your applications.
+:::
+
+:::{grid-item-card} {octicon}`paintbrush;2.5em;sd-mr-1` Styling
+:link: styling
+:link-type: doc
+
+In this section we will take you through the process of structuring a more complex application, discussing different approaches for managing reactivity and how to create a composable architecture for your applications.
```

Comment:
I think here is a copy & paste error - card description should be for styling tutorial?